Neos.Utility.ObjectHandling/Classes/ObjectAccess.php

```diff
@@ -376,23 +374,15 @@ public static function isPropertyGettable($object, $propertyName)
         if (!is_object($object)) {
             throw new \InvalidArgumentException('$object must be an object, ' . gettype($object) . ' given.', 1259828921);
         }
-        if ($object instanceof \ArrayAccess && isset($object[$propertyName]) === true) {
-            return true;
-        } elseif ($object instanceof \stdClass && array_search($propertyName, array_keys(get_object_vars($object))) !== false) {
+        if (($object instanceof \ArrayAccess && $object->offsetExists($propertyName)) || ($object instanceof \stdClass && array_key_exists($propertyName, get_object_vars($object)))) {
             return true;
         }
         $uppercasePropertyName = ucfirst($propertyName);
-        if (is_callable([$object, 'get' . $uppercasePropertyName])) {
-            return true;
-        }
-        if (is_callable([$object, 'is' . $uppercasePropertyName])) {
-            return true;
-        }
-        if (is_callable([$object, 'has' . $uppercasePropertyName])) {
+        if (is_callable([$object, 'get' . $uppercasePropertyName]) || is_callable([$object, 'is' . $uppercasePropertyName]) || is_callable([$object, 'has' . $uppercasePropertyName])) {
```

Comment:
I am neutral on that. Both work for me. Could also be refactored into a nice condition function at some point.